packages/E2ETest/app/ControlStyleTestPage.tsx

```diff
@@ -52,36 +58,56 @@ export function ControlStyleTestPage() {
   const onPressShowRoundBorder = () => {
     var previousState = showRoundBorder;
     setShowRoundBorder(!previousState);
- }
+  };
 
-   return (
+  return (
     <View>
       <View testID={'ControlStyleView'}>
-        {
-        /*
-        // Bug:4596 Switch
-        <Switch style={showRoundBorder? styles.roundBorder :styles.regularBorder} thumbColor='blue'/> 
-        */}
-        <CheckBox style={showRoundBorder? styles.roundBorder :styles.regularBorder} />
-        <TextInput style={showRoundBorder? styles.roundBorder :styles.regularBorder}
-          placeholder='TextBox'
-          placeholderTextColor='rgba(225,225,225,0.7)'
-          editable={false} />
+        <Switch
```

Comment:
This file got reformatted quite a bit since CI hasn't been enforcing our eslint file for this project. We should probably fix this, because VS Code will pick the file up and do it for you on save.